### DISCUSSION TITLE
Remove redundant syntax

### DISCUSSION THREAD

**Discussion by chihchenghuang** [tiangolo/uvicorn-gunicorn-docker#232] [Questions] [answered]
little things.
I noticed you used virtual package in the alpine dockerfile.
As I know, once you use this feature, you don't have to list all packages in the `apk del` command.
`apk del .build-deps` should work perfect here.

https://github.com/tiangolo/uvicorn-gunicorn-docker/blob/master/docker-images/python3.7-alpine3.8.dockerfile#L7

more discussions: 
https://stackoverflow.com/questions/46221063/what-is-build-deps-for-apk-add-virtual-command/46222036

Inpired a lot by your repo. good job man!

**Comment by tiangolo** [comment, marked as answer]
Thank you! And sorry for the very long delay. :sweat_smile: 

At some point I deprecated Alpine. :sweat_smile:  And now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image
